packages/shared/src/components/widgets/PostUsersHighlights.tsx

```diff
@@ -127,6 +128,7 @@ export const UserHighlight = (props: UserHighlightProps): ReactElement => {
     ('handle' in user ? user.handle : user.username) || id;
   const reputation = 'reputation' in user ? user.reputation : NaN;
   const companies = 'companies' in user ? user.companies : [];
+  const isPlus = 'isPlus' in user ? user.isPlus : false;
```

Comment:
Why do we do it so weird here vs:
```const isPlus = user?.isPlus;```

Reply:
<img width="592" alt="image" src="https://github.com/user-attachments/assets/4c430583-78e2-4bcc-a983-b7d6ed6d4304">
Strange type errors

Reply:
Ah, it's because `SourceAuthorProps` also has `Source` as a type :triggered:

Reply:
Rather check with typeof maybe? (just feels more verbose?)

Reply:
Not sure if it's possible, typescript will complain as long as it thinks that it doesn't exist on the type.

Reply:
<img width="597" alt="image" src="https://github.com/user-attachments/assets/22832b36-bcff-4194-865c-10649aa19926">

example

Reply:
Alterative is this
<img width="357" alt="image" src="https://github.com/user-attachments/assets/a32cf66f-fc07-47fb-bed6-1afb47327f96">

Reply:
Ah let's leave it then, not a big deal